packages/o/open-isns/pspec_x86_64.xml

```diff
@@ -0,0 +1,71 @@
+<PISI>
+    <Source>
+        <Name>open-isns</Name>
+        <Homepage>https://github.com/open-iscsi/open-isns</Homepage>
+        <Packager>
+            <Name>Jakob Gezelius</Name>
+            <Email>jakob@knugen.nu</Email>
+        </Packager>
+        <License>LGPL-2.1-or-later</License>
+        <PartOf>network.utils</PartOf>
+        <Summary xml:lang="en">iSNS server and client for Linux</Summary>
+        <Description xml:lang="en">iSNS server and client for Linux
+</Description>
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>open-isns</Name>
+        <Summary xml:lang="en">iSNS server and client for Linux</Summary>
+        <Description xml:lang="en">iSNS server and client for Linux
+</Description>
+        <PartOf>network.utils</PartOf>
+        <Files>
+            <Path fileType="config">/etc/isns/isnsadm.conf</Path>
+            <Path fileType="config">/etc/isns/isnsd.conf</Path>
+            <Path fileType="config">/etc/isns/isnsdd.conf</Path>
```

Comment:
Have you looked into what it would take to make these configs stateless?

Reply:
I had a check now, looked at arch and they had it the same. Fedora did not have this with a quick look.

Reply:
I don't know what you mean by that. It doesn't matter what Fedora and Arch do, we make our packages stateless